.circleci/config.yml

```diff
@@ -144,7 +144,7 @@ jobs:
       - run:
           name: Install and configure dependencies
           command: |
-            npm install -g gh-pages@2
+            yarn install
```

Comment:
Does switching from npm to yarn introduce any kind of whackiness?

Reply:
It's hard to test because this only runs on master. But we use `yarn` elsewhere so even if it does produce that kind of wackiness then I think it's worth it.